department-of-veterans-affairs/abd-vro
# Update README.md

## What was the problem?

Updated README.md with additional instructions for python install on Mac.
Typo Fixes

Associated tickets or Slack threads:
- #?

## How does this fix it?[^1]

Helps you install abd-vro without running into hiccups along the way.

## How to test this PR
- Step 1 : Setup python when performing local vro development setup
- Step 2[^secrel] Confirm Build Success
![Uploading Screenshot 2024-12-05 at 3.49.43 PM.png…]()



[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
